test/correctness/fuzz_simplify.cpp

```diff
@@ -266,96 +262,62 @@ bool test_expression(FuzzedDataProvider &fdp, Expr test, int samples) {
     return true;
 }
 
-// These are here to enable copy of failed output expressions
```

Comment:
Is there a reason for removing these? I found them to be useful for debugging when I've been debugging fuzz failures.

Reply:
They're a lot less necessary now that you can just use the same seed, and now that all subexpressions are tested too, so it's probably already reduced.